src/sentry/api/urls.py

```diff
@@ -613,108 +613,132 @@
+            name=f"{name_prefix}-group-reprocessing",
+        ),
+        re_path(
+            r"^(?P<issue_id>[^\/]+)/stats/$",
+            GroupStatsEndpoint.as_view(),
+            name=f"{name_prefix}-group-stats",
+        ),
+        re_path(
+            r"^(?P<issue_id>[^\/]+)/tags/$",
+            GroupTagsEndpoint.as_view(),
+            name=f"{name_prefix}-group-tags",
+        ),
+        re_path(
+            r"^(?P<issue_id>[^\/]+)/tags/(?P<key>[^/]+)/$",
+            GroupTagKeyDetailsEndpoint.as_view(),
+            name=f"{name_prefix}-group-tag-key-details",
+        ),
+        re_path(
+            r"^(?P<issue_id>[^\/]+)/tags/(?P<key>[^/]+)/values/$",
+            GroupTagKeyValuesEndpoint.as_view(),
+            name=f"{name_prefix}-group-tag-key-values",
+        ),
+        re_path(
+            r"^(?P<issue_id>[^\/]+)/(?:user-feedback|user-reports)/$",
+            GroupUserReportsEndpoint.as_view(),
+            name=f"{name_prefix}-group-user-reports",
+        ),
+        re_path(
+            r"^(?P<issue_id>[^\/]+)/attachments/$",
+            GroupAttachmentsEndpoint.as_view(),
+            name=f"{name_prefix}-group-attachments",
+        ),
+        re_path(
+            r"^(?P<issue_id>[^\/]+)/similar/$",
+            GroupSimilarIssuesEndpoint.as_view(),
+            name=f"{name_prefix}-group-similar",
+        ),
+        re_path(
+            r"^(?P<issue_id>[^\/]+)/external-issues/$",
+            GroupExternalIssuesEndpoint.as_view(),
+            name=f"{name_prefix}-group-external-issues",
+        ),
+        re_path(
+            r"^(?P<issue_id>[^\/]+)/external-issues/(?P<external_issue_id>\d+)/$",
+            GroupExternalIssueDetailsEndpoint.as_view(),
+            name=f"{name_prefix}-group-external-issues-details",
+        ),
+        re_path(
+            r"^(?P<issue_id>[^\/]+)/integrations/$",
+            GroupIntegrationsEndpoint.as_view(),
+            name=f"{name_prefix}-group-integrations",
+        ),
+        re_path(
+            r"^(?P<issue_id>[^\/]+)/integrations/(?P<integration_id>\d+)/$",
+            GroupIntegrationDetailsEndpoint.as_view(),
+            name=f"{name_prefix}-group-integration-details",
+        ),
+        re_path(
+            r"^(?P<issue_id>[^\/]+)/current-release/$",
+            GroupCurrentReleaseEndpoint.as_view(),
+            name=f"{name_prefix}-group-current-release",
+        ),
+        re_path(
+            r"^(?P<issue_id>[^\/]+)/first-last-release/$",
+            GroupFirstLastReleaseEndpoint.as_view(),
+            name=f"{name_prefix}-group-first-last-release",
+        ),
+        re_path(
+            r"^(?P<issue_id>[^\/]+)/participants/$",
+            GroupParticipantsEndpoint.as_view(),
+            name=f"{name_prefix}-group-participants",
+        ),
+        # Load plugin group urls
+        re_path(
+            r"^(?P<issue_id>[^\/]+)/plugins?/",
+            include("sentry.plugins.base.group_api_urls"),
```

Comment:
These URLs won't have control silo routes. I don't suspect that legacy plugins on legacy routes will be serving much traffic though.